larky/src/main/resources/vgs/http/request.star

```diff
@@ -55,6 +55,11 @@ def VGSHttpRequest(
         # call super init, with overrides
         self.__init__(url, data=data, headers=headers, method=method)
         self.url = url
+        parsed_url = parse.urlsplit(url)
```

Comment:
what if this fails?

Reply:
Or can we just suggest running this code in Larky 🤔

Reply:
i'm pretty sure this is guaranteed to succeed if we've already gotten this far, since we do the exact same thing inside of __init__

Reply:
and yeah, this is effectively a convenience, but it's what's being asked for in the ticket